boot.php

```diff
@@ -153,7 +153,7 @@
         }, rex_extension::LATE);
     } else {
         /* Undo-Action will only last for one page reload */
-        if ($deleteQueue && (time() - rex_session('undo_timestamp')) >= 30 && !$isApiCall) {
+        if ($deleteQueue && rex_session('undo_timestamp') && (time() - rex_session('undo_timestamp')) >= 30 && !$isApiCall) {
```

Comment:
vermutlich geht auch (ungetestet)

```suggestion
        if ($deleteQueue && (time() - rex_session('undo_timestamp', 'int')) >= 30 && !$isApiCall) {
```

Reply:
Dann müsste der Wert aber Default zumindestens 0 sein, oder wäre es durch den cast 0 ?

Reply:
soweit ich mich erinnere ist der default dann automatisch `0`, da man beim aufruf gesagt hat, dass man einen `int` will

Reply:
habs getestet, funzt.